test/settings/DynamoSettings-NewSettings.xml

```diff
@@ -32,6 +32,7 @@
   <WindowW>1936</WindowW>
   <WindowH>1056</WindowH>
   <UseHardwareAcceleration>false</UseHardwareAcceleration>
+  <HomePageSettingsSerialized>{"recentPageViewMode":"grid","samplesViewMode":"list"}</HomePageSettingsSerialized>
```

Comment:
@dnenov Why do we serialize the setting as a json block instead of xml list? This is slightly different than other Dynamo settings

Reply:
Hi @QilongTang - I am glad you cast a glance at this! What I am trying to achieve is minimize the amount of separate setting variables we introduce for the HomePage while future proofing the process of introducing new settings. With that idea, I wanted to use a Dictionary<string, obj> (name of the property, value of the property) - as we don't know what type of value a certain setting might have. This type of dictionary is not directly serializable though, so I used a slight workaround to go through .json serialized string. 

I think it's the 'object' part that did not allow me to serialize to xaml directly.

Reply:
In hindsight, a Dictionary<string, string> might have worked just fine, at least for the time being.

Let me know what you think, I am more than happy to rework that bit of code to make things less complicated if possible!

Reply:
I am leaning towards a Dictionary<string, string> instead so we can serialize to xml, so that we can stick to XML. In terms of the specific setting, if casting to bool or double is required in the future, we can do that. I do appreciate your thinking on the future proof though, but we also have customers manually create settings template for the whole company so I would like to prioritize consistency.

Reply:
Thanks Aaron. Discussed this with Deyan and he will make that change in a new PR. Will go ahead and merge this.